docs_outdoornav_user_manual/api/api_endpoints/definitions.mdx

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
@civerachb-cpr  Should all the changes in these docs_outdoornav_user_manual files be included in the docs for the 0.9.0 release? If so, just want to make sure we get this all merged in prior to @dobrienCP creating the 0.9 release for the OutdoorNav docs.

Reply:
Discussed offline with Chris and confirmed that we want these changes.